app/models/school.rb

```diff
@@ -18,8 +18,16 @@ class School < ApplicationRecord
   has_many :early_career_teacher_profiles
   has_many :early_career_teachers, through: :early_career_teacher_profiles, source: :user
 
+  scope :with_name, lambda { |search_key|
```

Comment:
Is there a name for this that makes it more obvious that this matches part of the name? This sounds like `find_by_name` at the moment (to me at least)

Reply:
Maybe just `with_name_like`?

Reply:
Good point - changed